Cockpit 340 (#802)
cockpit-project/cockpit-project.github.io: _posts/2025-06-04-cockpit-340.md

---
title: Cockpit 340
author: Venefilyn
date: "2025-06-04"
tags: cockpit, podman, files, machines, ostree
slug: cockpit-340
category: release
summary: "Handle unsupported storage partitions, link quadlet logs to their service, source mode for virtual networks, various UX improvements."
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly.

Here are the release notes from Cockpit 340, cockpit-podman 107, cockpit-machines 333, and cockpit-ostree 211:

## Storage: Prevent modifying partitions in unsupported places

> It was previously confusing and somewhat dangerous when storage page listed LVM2 Logical Volumes as having free space. This is now changed to include a notice and not display these partitions to prevent mishaps when we cannot support it.

Reviewer: This is really *really* hard to understand. After talking to Freya a bit, this is the 50% that I understand:

```suggestion
It was previously confusing and somewhat dangerous when the Storage page listed LVM2 Logical Volumes as having free space. That is an internal implementation detail, and acting on that could have dangerous consequences. The page now does not display these partitions any more.
```

But honestly this sounds more like a bug fix, and just dropping that topic altogether would be less confusing altogether IMHO.

![Screenshot showing off a notice inside a GPT partitions card saying "Unexpected partitions" in bold, followed by a description saying "Partitions are not supported on this block device. If it is used as a disk for a virtual machine, the partitions must be managed by the operating system inside the virtual machine."]({{ site.baseurl }}/images/340-storage-unsupported-partitions.png)

## UX improvements

- We now warn the user when renaming volume groups that contain the root filesystem as it might break the next boot.

> - Health card items in overview page is now aligned correctly.

Reviewer: These are completely unrelated. The second one is just a minor bug fix, and should IMHO be dropped. The remaining one could be "Storage: Rename safety warning".

  _Thanks to [Albert Byiringiro](https://github.com/albert-byiringiro) for this contribution!_

  ![Screenshot of the overview page and health card with the items inside it aligned.]({{ site.baseurl }}/images/340-health-card-spacing.png)

> ## Podman: Quadlets logs now links to their systemd service log page

Reviewer: ```suggestion
## Podman: Quadlet log links
```

> A simple but quality-of-life improvement that adds a link from the Logs page in a Quadlet container to its systemd service counterpart.

Reviewer: ```suggestion
Quadlet logs now link to their systemd service log page counterpart.
```

---
title: Cockpit 340
author: Venefilyn
date: "2025-06-04"
tags: cockpit, podman, files, machines, ostree
slug: cockpit-340
category: release
summary: "Handle unsupported storage partitions, link quadlet logs to their service, source mode for virtual networks, various UX improvements."
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly.

Here are the release notes from Cockpit 340, cockpit-podman 107, cockpit-machines 333, and cockpit-ostree 211:

## Cockpit

### Storage: Prevent modifying partitions in unsupported places

It was previously confusing and somewhat dangerous when storage page listed LVM2 Logical Volumes as having free space. This is now changed to include a notice and not display these partitions to prevent mishaps when we cannot support it.

> ![Screenshot showing off a notice inside a GPT partitions card saying "Unexpected partitions" in bold, followed by a description saying "Partitions are not supported on this block device. If it is used as a disk for a virtual machine, the partitions must be managed by the operating system inside the virtual machine."]({{ site.baseurl }}/images/340-storage-unsupported-partitions.png)

Reviewer 2: This screenshot is in dark mode while the rest are in light mode. As we don't have any way to switch dark/light mode screenshots we should keep them in light mode as that is the default.

Reviewer: s/off//

### UX improvements

- We now warn the user when renaming volume groups that contain the root filesystem as it might break the next boot.
- Health card items in overview page is now aligned correctly.

  _Thanks to [Albert Byiringiro](https://github.com/albert-byiringiro) for this contribution!_

  ![Screenshot of the overview page and health card with the items inside it aligned.]({{ site.baseurl }}/images/340-health-card-spacing.png)

> - Various typos have been fixed.

Reviewer 2: We usually included new features not small UX / spelling mistakes
